README.md

```diff
@@ -237,9 +237,22 @@ Run tests using `rake`. The default task runs all the tests.
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create new Pull Request
 
+## Docker
+
+We've provided a simple docker image that can be used to test litmus_paper, run tests, or to assist in feature development.
+
+To run it, from the top of your copy of the litmus_paper repository:
+
+```
+docker build -t litmus_paper .
+docker run --rm -it -v $(pwd):/home/litmus_paper -P litmus_paper /bin/bash
```

Comment:
Instead of running docker with bash I think you want to set the `CMD`  in the dockerfile to be the litmus command.  Instead of bind mounting litmus, I think it's better to `ADD` it to the docker container in the dockerfile.  That way, the user can just `docker run` it without the extra pieces.